packages/features/ee/teams/components/AddNewTeamMembers.tsx

```diff
@@ -229,7 +251,7 @@ const PendingMemberItem = (props: { member: TeamMember; index: number; teamId: n
           <div className="flex space-x-1">
             <p>{member.name || member.email || t("team_member")}</p>
             {/* Assume that the first member of the team is the creator */}
-            {index === 0 && <Badge variant="green">{t("you")}</Badge>}
+            {member.id === session.data?.user.id && <Badge variant="green">{t("you")}</Badge>}
```

Comment:
Now that belonging to a team is optional, `index === 0` member could be someone else